apps/showcase/src/app/placeholder/placeholder.component.ts

```diff
@@ -0,0 +1,67 @@
+import { AsyncPipe } from '@angular/common';
+import { type AfterViewInit, ChangeDetectionStrategy, Component, type QueryList, ViewChildren, ViewEncapsulation } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { O3rComponent } from '@o3r/core';
+import { PlaceholderRulesEngineActionHandler, PlaceholderRulesEngineActionModule } from '@o3r/components/rules-engine';
+import { DynamicContentModule, DynamicContentService } from '@o3r/dynamic-content';
+import { RulesEngineDevtoolsModule, RulesEngineRunnerModule, RulesEngineRunnerService, RulesetsStore, setRulesetsEntities } from '@o3r/rules-engine';
+import { Store } from '@ngrx/store';
+import { firstValueFrom } from 'rxjs';
+import { IN_PAGE_NAV_PRES_DIRECTIVES, type InPageNavLink, InPageNavLinkDirective, InPageNavPresService, PlaceholderPresComponent } from '../../components';
+import { environment } from '../../environments/environment.development';
+import { TripFactsService } from '../../facts';
+
+@O3rComponent({ componentType: 'Page' })
+@Component({
+  selector: 'o3r-placeholder-page',
+  standalone: true,
+  imports: [
+    PlaceholderPresComponent,
+    DynamicContentModule,
+    RulesEngineRunnerModule,
+    RulesEngineDevtoolsModule,
+    PlaceholderRulesEngineActionModule,
+    RouterModule,
+    IN_PAGE_NAV_PRES_DIRECTIVES,
+    AsyncPipe
+  ],
+  templateUrl: './placeholder.template.html',
+  styleUrl: './placeholder.style.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlaceholderComponent implements AfterViewInit {
+  @ViewChildren(InPageNavLinkDirective)
+  private readonly inPageNavLinkDirectives!: QueryList<InPageNavLink>;
+  public links$ = this.inPageNavPresService.links$;
+
+  constructor(
+    private readonly inPageNavPresService: InPageNavPresService,
+    private readonly tripFactsService: TripFactsService,
+    private readonly store: Store<RulesetsStore>,
+    private readonly dynamicContentService: DynamicContentService,
+    rulesEngine: RulesEngineRunnerService,
+    placeholderRulesEngineActionHandler: PlaceholderRulesEngineActionHandler
+  ) {
+    this.tripFactsService.register();
+    rulesEngine.actionHandlers.add(placeholderRulesEngineActionHandler);
+    void this.loadRuleSet();
+  }
+
+  private async loadRuleSet() {
+    const path = await firstValueFrom(
```

Comment:
__THOUGHT__
We have to places where we load the same 'rulesets.json' file. 
Would not be better to load it once at app level?
I am just trying to see how it could be interpreted by someone looking for an example of rules engine usage. They will load it as we do, at route level.
From what I remember from the guidelines, the rulesets are loaded once at app level and linked to components if we don;t want them to be active from the beginning. 
Opinions ?

Reply:
I think the idea behind was to keep the app component as simple as possible and to add the logic in each page to understand what is needed for this feature.
I agree that it could be confusing we can discuss that with the team :)

Reply:
As discussed we will add a comment to explain that for the sake of example we put it in the page logic but usually we recommend to set it in the app component